assets/js/main.js

```diff
@@ -43,3 +47,53 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
     left: -9999px;
     `;
 }));
+
+// Form Validation
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const formMessage = ['Email should be in lower case', 'Submition aproved'];
+  const emailAddres = email.value;
+  if (emailAddres.toLowerCase() !== emailAddres) {
+    validationMsg.classList.remove('contact-button-success-message');
+    validationMsg.classList.add('contact-button-message');
+    validationMsg.innerHTML = `${formMessage[0]}`;
+  } else {
+    validationMsg.innerHTML = `${formMessage[1]}`;
+    validationMsg.classList.add('contact-button-success-message');
+  }
+});
```

Comment:
[REQUIRED] An excellent work on validating the input format for the `email field` :clap:  However, since you `prevented the submission` as soon as the function invokes, it is not submitting the form even with the correct format (all lower case), it is nicely presenting the message saying that "Submission approved" though (refer to the screenshot below):
![Screenshot from 2022-04-07 22-56-54](https://user-images.githubusercontent.com/67608876/162267806-849f5fd7-337a-4f47-920f-d138e650f641.png)

- [ ] Please kindly make sure that the form is actually submitted if the input format is correct.

_**hint**_
The reason behind the issue is that `e.preventDefault()` is at a higher scope compared to the scopes of `conditional statements` meaning that regardless of the condition, the function prevents the form from submission, You can solve this issue by just placing the `e.preventDefault()` within a conditional statement with the following logic:
```JAVASCRIPT
// if format is not correct
//   prevent from submission
//   display err message
// you do not even need else statement unless you want to display success message
```